packages/airnode-deployer/src/utils/files.ts

```diff
@@ -60,11 +61,47 @@ export function parseReceiptFile(receiptFilename: string) {
     throw e;
   }
 
-  const validationResult = validateReceipt(receipt);
+  const validationResult = validateReceipt(receipt, nodeVersion);
   if (!validationResult.valid) {
     logger.fail('Failed to validate receipt file');
     throw new Error(`Invalid Airnode receipt file: ${JSON.stringify(validationResult.messages)}`);
   }
 
   return receipt as Receipt;
 }
+
+export function parseConfig(configPath: string, secrets: Record<string, string>): Config {
+  let rawConfig;
+
+  try {
+    rawConfig = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+  } catch (e) {
+    logger.fail('Failed to parse config file');
+    throw e;
+  }
+
+  const result = validateJsonWithTemplate(rawConfig, `config@${getNodeVersion()}`, secrets, true);
+  const config = result.specs as Config;
+
+  if (config.nodeSettings.skipValidation) {
+    logger.warn('Skipping the config validation');
+    return config;
+  }
+
+  if (config.nodeSettings.cloudProvider.type === 'local') {
+    const message = "Deployer can't deploy to 'local' cloud provider";
+    logger.fail(message);
+    throw new Error(message);
+  }
```

Comment:
Skipping the validation is mainly useful for development when you want to try new features regardless of the current released version. But I don't think skipping this cloud provider test is useful in any way. I would keep this test even when the `skipValidation` flag is set.